### ISSUE TITLE
Crash on file move (v1.7.5)

### ISSUE THREAD

**Issue by aubreyz** [gsantner/markor#550]
#### General information

* **App version:** 1.7.5 (67)
* **System:** Moto g6

#### Description
There is a crash on attempted file move within the Markor Documents folders. This is a regular crash (every time) with different files and folders and on two phones. No problem moving files using an independent file manager.  Crash is immediately on selecting "select this folder"

Greatest note/markdown manager for Android by the way.

![crash](https://user-images.githubusercontent.com/25100168/56865160-9ec82700-69c2-11e9-94e7-af4219cda3c5.jpg)

**Comment by gsantner** [comment]
can you also confirm this issue happens on 1.7.6?

**Comment by aubreyz** [comment]
Not sure how to install 1.7.6. I can only install from playstore on current phone and that is still at 1.7.5. Happy to do if guided.

**Comment by gsantner** [comment]
The update should be available today or tomorrow on f-droid.
you can also get the update from here <https://github.com/gsantner/markor/releases>, it is signed with the google play key.

**Comment by aubreyz** [comment]
Yes same crash on 1.7.6 (google play version)

**Comment by aubreyz** [comment]
1.7.7 (69) still this issue.

**Comment by gsantner** [comment]
Can you try 1.8.0?

**Comment by aubreyz** [comment]
1.8.0 file copies fine. However there is a new issue now (at least on the one time I tried) - no time to experiment more now: 

I moved file from root folder to a subfolder - it worked fine. However if, immediately after doing the copy, I try to navigate into the folder to which file was moved, it selects rather than opens that folder. The only way to open the folder was to close and restart Markor. If the issue remains I'll add some screenshots. However the file move now seems fixed. Many thanks for all your work.
